Gemfile

```
@@ -18,6 +18,7 @@ gem "pry"
 gem "rack-cors"
 gem "representable"
 gem "multi_json"
+gem "config"
```

rubocop (1.68.0)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `config` should appear before `multi_json`.

(Bundler/OrderedGems)